course_content/notebooks/dask_intro.ipynb

```diff
@@ -728,13 +728,364 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": true
+    "collapsed": false
    },
    "outputs": [],
    "source": [
     "print(delayed_merged_cube.compute())"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using Iris Load with Dask and Constraints"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Often when loading files in Iris we will want to constrain the files before loading them. Loading using constraints has already been covered in the Iris Introduction course and in the Iris documentation. We will now consider some simple examples of loading with constraints and how to use dask in this process."
```

Comment:
Why? That is, why might we want to constrain the files? I guess it should be clear to most why we would want to, but I think a reason for the statement you've made here would help the flow.

Also, technically you constrain _on load_ rather than _before load_, so perhaps you could make the end of the first sentence here something like:

```...constrain the files on load, in order to <reason for constraining>.```